apps/experiments/models.py

```diff
@@ -356,6 +355,11 @@ class Experiment(BaseTeamModel):
         null=True,
         blank=True,
     )
+    max_token_limit = models.IntegerField(
+        default=0,
```

Comment:
this default also needs to be updated to "8192"

Reply:
I kept this as 0 so that new experiments can set it to what they want. I would like set the default on the UI based on the model being used but that was too complicated for this PR.

Reply:
Wouldn't the discrepancy between this default value and the one in the migration cause a new migration to be generated that updates the default value then?

Reply:
You are correct. For some reason I thought you could do this 🤷🏻‍♂️ 1f1a369